plone/portlet/static/static.py

```diff
@@ -17,15 +20,38 @@
 
 logger = logging.getLogger('plone.portlet.static')
 PLONE5 = getFSVersionTuple()[0] >= 5
+USE_AUTOFORM = IAutoExtensibleForm.implementedBy(base.AddForm) or PLONE5
+try:
+    import plone.app.widgets  # noqa
+    USE_PAE = True
```

Comment:
@gforcada good idea, done.  Dropped two (revert) commits, squashed the others.  Checks pass.

CC: @petri